### ISSUE TITLE
Regularization on primitives 

### ISSUE THREAD

**Issue by moshanATucsd** [zekunhao1995/DualSDF#4]
In [regularization](https://github.com/zekunhao1995/DualSDF/blob/5882f16cdc41c149cab86215756595de062640fc/trainers/dualsdf_trainer.py#L171), square root is not used in `dists = torch.sum(attrs[:,:,1:]**2, dim=-1, keepdim= True)`, but it seems this is adding a regularization on the center of primitive, which should locate with in a sphere with radius 1.05. So I am wondering why square root is not used, as in [this line](https://github.com/zekunhao1995/DualSDF/blob/5882f16cdc41c149cab86215756595de062640fc/models/sdfsphere.py#L25)?

**Comment by zekunhao1995** [comment]
Hi Mo,
Good question. You are right that the regularization here is to prevent the primitives from leaving the unit sphere. The type of regularization function here is not very critical so we have went for the simplest, which essentially penalizes the squared distance between the center of the spheres and the origin. Although I haven't tried, adding back the square root should also work.
Best

**Comment by moshanATucsd** [comment]
Got it, thanks!
